entries/c/collegeboard.org.json

```diff
@@ -2,10 +2,10 @@
   "College Board": {
     "domain": "collegeboard.org",
     "url": "https://www.collegeboard.org/",
-    "contact": {
-      "facebook": "thecollegeboard",
-      "twitter": "collegeboard"
-    },
+    "tfa": [
+      "email"
+    ],
+    "documentation": "https://my.collegeboard.org/profile/information",
```

Comment:
The documentation page can't be behind a login wall.
```suggestion
```

Reply:
There is no documentation though. The only mention of MFA is a switch (see #6692 for a screenshot), and I figured out the email part by signing in.

Reply:
So remove the documentation element as I suggested.

Reply:
Didn't see that, sorry! Thank you!